app/views/execution_environments/_form.html.slim

```diff
@@ -51,6 +58,5 @@
     = f.select(:testing_framework, @testing_framework_adapters, {include_blank: true}, class: 'form-control')
   .mb-3
     = f.label(:help, class: 'form-label')
-    = f.hidden_field(:help)
-    .form-control.markdown
+    = f.markdown :help
```

Comment:
I assume that this `help` field was hidden because the Ace editor logic added a markdown input field, for which the result has been sent as params to the controller anyway, and thus we did not write `:help` directly. With the new `markdown` type, we can associate the field directly.

Is my assumption correct?

Reply:
The [markdown type](https://github.com/openHPI/codeocean/pull/2242/files#diff-774edc8b4e4b0676d71b60c674548defcf8312cae33d6a9b314ae1286665b1bb) builds a hidden textarea (in this case hidden with a bootstrap class, but could have also been done using a `hidden_field` element) associated with `:help` as well as an element the javascript uses to build the editor. On every editor change the [textarea is updated](https://github.com/openHPI/codeocean/pull/2242/commits/841d32605b46f6ce8ce6980cc0ee8c23cba108d4#diff-38f3d41ff9f2935a8a00af376a734596bc6748078880f054f4ffb35ff5c93493R40) accordingly. 

I think this was also essentially [the same we did with the Ace editor](https://github.com/openHPI/codeocean/pull/2242/commits/90256c39f5466f10d6d36930336de3e20ce23416#diff-65b0597ab46cb985c89d05227dcd0e3f073f8c2868da51ce6e874d1b6313d209L4-L5), but without having a FormBuilder.